swan-pipeline/src/main/java/de/fraunhofer/iem/swan/cli/CliRunner.java

```diff
@@ -15,7 +15,7 @@ public class CliRunner implements Callable<Integer> {
     private String testDataDir = "";
 
     @CommandLine.Option(names = {"-train", "--train-data"}, description = {"Path of training JARs or class files"})
-    private String trainDataDir = "";
+    private String trainDataDir = "/Users/rohith/Desktop/IEM/Datasets-New/Jar";
```

Comment:
Please undo these changes so that the configuration in this file remains unchanged.